bool path_choice
string controller_method
bool stop_flag
bool blind_rush
bool park_mode
bool is_autonomous